config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-external-ca.yaml

```diff
@@ -0,0 +1,162 @@
+# periodic jobs
+
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kinder-external-ca-master
```

Comment:
```suggestion
- name: ci-kubernetes-e2e-kubeadm-kinder-external-ca-latest
```

Reply:
`master` is being renamed to `main`. shouldn't we use that?

Reply:
@fabriziopandini how about we include this as `master` and send separate PRs to rename all jobs as per?
https://github.com/kubernetes/kubeadm/issues/2302